patches/services-network-url_loader.cc.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/services/network/url_loader.cc b/services/network/url_loader.cc
+index c9e8660da7abc1dda24720457aa006299ad6fe4e..1a0ebabb5b6d4634b7e94f1be1d8b07e7b620cd6 100644
+--- a/services/network/url_loader.cc
++++ b/services/network/url_loader.cc
+@@ -516,8 +516,8 @@ URLLoader::URLLoader(
+              !request.trusted_params->isolation_info.IsEmpty()) {
+     url_request_->set_isolation_info(request.trusted_params->isolation_info);
+     if (request.credentials_mode != network::mojom::CredentialsMode::kOmit) {
+-      DCHECK(url_request_->isolation_info().site_for_cookies().IsEquivalent(
```

Comment:
@iefremov do we need to update site for cookies when doing static redirect?

Reply:
I believe neither us nor Chromium should do this.. How do we hit the DCHECK?

Reply:
we basically should reflect [1] in `BraveProxyingUrlLoaderFactory`, I'll push

[1] https://source.chromium.org/chromium/chromium/src/+/7d4a3137acc42de817a8f143479b0de47ffd02f5:extensions/browser/api/web_request/web_request_proxying_url_loader_factory.cc;dlc=86aeca3688cbee41480bc53a81e12ed379c76e7d?originalUrl=%2F

Reply:
@mkarolin https://github.com/brave/brave-core/pull/5679/commits/676b0f33ab30ca1eb6cf6aeca6e77922e2a9ff8d

Reply:
@iefremov I am still getting the DCHECK :(
STR:
1. Start browser
2. Navigate to a site in a different from your language setting language (e.g. https://www.lemonde.fr/)
3. Wait for the translation bubble suggesting to install Google Translate to show up.
4. Click Install
5. In the extension installation bubble click Add extension 
6. DCHECK